### DISCUSSION TITLE
Matterbridge helper-script needs to be corrected

### DISCUSSION THREAD

**Discussion by ToRa89** [tteck/Proxmox#3498] [Help Needed] [answered]
Hey there,
I testet be helptger-script for Matterbridge and had to open and issue there. 
The result was the issue on the Helper-Script config.
I’ll link to the other issue with further information from the dev.

**Comment by ToRa89** [comment]
https://github.com/Luligu/matterbridge-zigbee2mqtt/issues/50#issuecomment-2277559621

**Comment by tteck** [comment]
I have no matter devices to test. So, @MickLesk, could you take a look at this, since it's your PR? https://github.com/tteck/Proxmox/pull/3178

**Reply by MickLesk** [reply]
@weekend, im not home - i will Check with my nuki matter device

**Reply by tteck** [reply]
Any news?

**Comment by tteck** [comment, marked as answer]
~🧐 As this [issue](https://github.com/tteck/Proxmox/issues/3499) is unrelated to the script, it is considered closed.~

Fixed.
